revolv/templates/base/home.html

```diff
@@ -54,10 +59,49 @@
             <div class="learn-more button">Learn how RE-volv works</div>
         </a>
     </div>
+</div> -->
+
+<div class="cover">
+    <a target="_blank" href="https://youtu.be/eSADOAxjcPU">
+    <div class="shade"></div>
+    <div class="cover-inset">
+        <h1>WE'RE SAVING TOMORROW</h1>
+        <p>When you donate to RE-volv's solar projects, you are making a lasting impact on the environment and the world around you.</p>
+        <a href="#how-revolv-works">
+            <div class="learn-more button">Learn how RE-volv works</div>
+        </a>
+    </div>
+    <video autoplay loop muted src="{% static "videos/cover.mp4" %}" class="cover-video"></video>
+    </a>
+</div>
+
+{% if first_project %}
+<div class="text-with-divider">OUR CURRENT PROJECT</div>
```

Comment:
This line, the next line, and the line after should be increasing levels of indentation :)